docs: update guidance for triager nominations (#6349)
expressjs/express: Contributing.md

a new triager. If you are interested in becoming a triager, our best advice is to actively participate
in the community by helping triaging issues and pull requests. As well we recommend
to engage in other community activities like attending the TC meetings, and participating in the Slack

> discussions. If you feel ready, first reach out to an active member of the organization to ask if they'd

Reviewer: ```suggestion
discussions. If you feel ready and have been helping triage some issues, reach out to an active member of the organization to ask if they'd
```

Author: i like it!

> be willing to support you. If they agree, they can create a pull request to formalize your nomination

Reviewer 2: ```suggestion
be willing to support you. If they agree, they can create a pull request to formalize your nomination. In the case of an objection to the nomination, the PR Creator is responsible for working with the individuals involved and finding a resolution.
```

It is important that in case of an objection to ensure that we help the nominee while possible, otherwise the nominees might feel unwelcome to the project :thinking:.

BTW... Node has a similar policy for certain roles like Core Team ([ref](https://github.com/nodejs/node/blob/main/GOVERNANCE.md?plain=1#L173))

Author: This is important and was not considered before. Here, I think it should be the triage group instead of the person who creates the PR.

Author: ```suggestion
be willing to support you. If they agree, they can create a pull request to formalize your nomination. In the case of an objection to the nomination, the triage team is responsible for working with the individuals involved and finding a resolution.
```
